lib/rules/no-invalid-regexp.js

```diff
@@ -62,4 +72,8 @@ module.exports = function(context) {
 
 };
 
-module.exports.schema = [];
+module.exports.schema = [{
```

Comment:
Could you change schema to:
```js
{
  "type": "object",
  "properties": {
    "allowConstructorFlags": {
      "type": "array",
      "items": {
        "type": "string"
      }
    }
}
```
It's more descriptive and will catch more issues.